sw/runtime/src/omp/omp.c

```diff
@@ -5,8 +5,9 @@
 #include <string.h>
 
 #include "omp.h"
-
+#ifdef SNRT_SUPPORTS_DMA
```

Comment:
See comments in `start.c`, then this, and other changes in file, should no longer be necessary.